package.json

```diff
@@ -2,25 +2,46 @@
   "name": "single-spa",
   "version": "5.9.3",
```

Comment:
If this is the 6 branch, may be worth updating this right now too?

Reply:
I only update version during publish, since `pnpm version major` automatically updates package.json